lib/internal/dns/utils.js

```diff
@@ -131,9 +130,8 @@ class Resolver {
   setLocalAddress(ipv4, ipv6) {
     validateString(ipv4, 'ipv4');
 
-    if (typeof ipv6 !== 'string' && ipv6 !== undefined) {
-      throw new ERR_INVALID_ARG_TYPE('ipv6', ['String', 'undefined'], ipv6);
-    }
+    if (ipv6 !== undefined)
+      validateString(ipv6, 'ipv6');
```

Comment:
Doesn't this change the error message, which previously suggested `String` _or_ `undefined`?

Reply:
It kinda does, but we mostly don't mention if a value needs to be `undefined` in almost all of the validation across the internals, and the `String` seems out of place and inconsistent with the other validations.

Reply:
We can mark it as semver-major to be extra-sure.

Reply:
We can, but the point of error codes is that we don't need to consider error message changes to be semver-major.